integration-tests/browser/package.json

```diff
@@ -5,7 +5,7 @@
     "license": "Apache-2.0",
     "private": true,
     "scripts": {
-        "build": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js",
+        "build": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --mode=development",
```

Comment:
I was seeing the integration tests occasionally timeout in CI and this seems to help get them passing.